util/dvsim/dvsim.py

```diff
@@ -444,18 +478,7 @@ def parse_args():
                      help=("Print dvsim tool messages but don't actually "
                            "run any command"))
 
-    return parser.parse_args()
-
-
-def main():
-    args = parse_args()
-
-    # We want the --list argument to default to "all categories", but allow
-    # filtering. If args.list is None, then --list wasn't supplied. If it is
-    # [], then --list was supplied with no further arguments and we want to
-    # list all categories.
-    if args.list == []:
-        args.list = _LIST_CATEGORIES
```

Comment:
This code got duplicated by a fluffed rebase in commit b7aacc1: there's another copy a couple of lines lower down.